taichi/rhi/cpu/cpu_device.h

```diff
@@ -49,6 +49,8 @@ class CpuCommandList : public CommandList {
   }
 
   void bind_pipeline(Pipeline *p) override{TI_NOT_IMPLEMENTED};
+  void set_push_constants(size_t constants_size,
+                          void *constants_data) override{TI_NOT_IMPLEMENTED};
```

Comment:
```suggestion
                          void *constants_data) override{TI_NOT_IMPLEMENTED;};
```

This coding style looks a bit weird...

Reply:
That's an oops